Tests/test_file_apng.py

```diff
@@ -440,6 +440,12 @@ def test_apng_save_duration_loop(tmp_path):
         assert im.n_frames == 1
         assert im.info.get("duration") == 750
 
+    # test removal of duplicated frames with a single duration
+    frame.save(test_file, save_all=True, append_images=[frame, frame], duration=500)
+    with Image.open(test_file) as im:
+        assert im.n_frames == 1
+        assert im.info.get("duration") == 1500
```

Comment:
`test_apng_save_duration_loop` is testing quite a few things that don't depend on each other, can we at least move this chunk into its own test case? 

For example, it could be useful to know if this bit passes, even if earlier bits fail.